src/Main.elm

```diff
@@ -1397,25 +1361,22 @@ decodePresence : Json.Decode.Decoder Presence
 decodePresence =
     Json.Decode.oneOf
         [ decodePresenceString
+
+        -- for backwards compatibility
         , Json.Decode.field "type" Json.Decode.string
             |> Json.Decode.andThen
                 (\type_ ->
                     case type_ of
                         "System" ->
```

Comment:
decoding presence is now
- our new json is just a string, so prefer `decodePresenceString`
- if that didn't work, we try decoding as object and looking for the `type` (this is based on old json)